Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM quay.io/ecosystem-appeng/keycloak-source:latest as keycloak
+
+FROM registry.access.redhat.com/ubi9/openjdk-11-runtime
```

Comment:
Why did you pick this image? 
i don't think it's the latest

Reply:
i did not chose it. i Reused from 
https://github.com/dmartinol/keycloak-change-admin-kcadm/tree/main/docker

LMK if you want me to go with latest

Reply:
latest compatible